supabase/migrations/20240223012557_sentences.sql

```diff
@@ -0,0 +1,182 @@
+CREATE TABLE sentences (
+  id uuid unique primary key NOT NULL, -- generated on client so users can create a sentence offline and keep editing it
+  "text" text,
+  translation jsonb,
+  created_at timestamp with time zone DEFAULT now() NOT NULL,
+  created_by uuid NOT NULL REFERENCES auth.users (id),
+  updated_at timestamp with time zone DEFAULT now() NOT NULL,
+  updated_by uuid NOT NULL REFERENCES auth.users (id),
+  deleted timestamp with time zone
+);
+
+ALTER TABLE sentences ENABLE ROW LEVEL SECURITY;
+
+CREATE TABLE senses_in_sentences (
+  sense_id uuid NOT NULL REFERENCES senses (id),
+  sentence_id uuid NOT NULL REFERENCES sentences (id),
+  created_by uuid NOT NULL REFERENCES auth.users (id),
```

Comment:
I think I understand the reason behind this table: having a many to many relation between senses and sentences.
But not sure about the reason why this table has the fields created_by and updated_at too.
Is it due to the relationship can be updated? like changing a sentence from sense 2 to another sense like sense 4 for example? @jacob-8

Reply:
Yes relationships can be added and removed from any sense to any sense. We want to know who made the change and when.

Reply:
Ok, Thanks!